src/TeamSpeak3QueryApi/Specialized/Responses/DBIDFromUID.cs

```diff
@@ -0,0 +1,12 @@
+namespace TeamSpeak3QueryApi.Net.Specialized.Responses
+{
+    public class DBIDFromUID : Response
+    {
+        [QuerySerialize("cluid")]
+        public string cluid;
```

Comment:
We should be consistent with the existing wording and casing:
https://github.com/nikeee/TeamSpeak3QueryAPI/blob/c9127c526aeccbeff28bf115ce6ad56e74df359a/src/TeamSpeak3QueryApi/Specialized/Notifications/TokenUsed.cs#L5-L12